warehouse/packaging/models.py

```diff
@@ -107,6 +112,13 @@ def __getitem__(self, version):
             raise KeyError from None
 
     @property
+    def latest_release(self):
+        try:
+            return self.releases.order_by('created')[0]
```

Comment:
`[0]` implies `.limit(1)`, I'll add a test to cover the case when there are no releases and rename the method